pkg/kubelet/images/image_manager.go

```diff
@@ -224,3 +308,68 @@ func applyDefaultImageTag(image string) (string, error) {
 	}
 	return image, nil
 }
+
+// isEnsuredBySecret - returns two bools
+// 1. pulledBySecret: true if it is in ensured secret pulled image list,
+// the list will clean the image after PullImageSecretRecheckDuration
+// 2. ensuredBySecret: true if the secret for an auth used to pull an
+// image has already been authenticated through a successful pull request
+// and the same auth exists for this podSandbox/image.
+func (m *imageManager) isEnsuredBySecret(imageRef string, image kubecontainer.ImageSpec, pullSecrets []v1.Secret) (pulledBySecret, ensuredBySecret bool) {
+	m.lock.Lock()
```

Comment:
This function ignores all the errors. We should either log the error or (probably better) return the error so the caller can log it.

Reply:
Add an err return, and log the err.

Reply:
@pacoxu I see that comment has been resolved, but the code in that function still neither logs nor returns the error.